### DISCUSSION TITLE
Did anyone port this yet to Android?

### DISCUSSION THREAD

**Discussion by adewilt8** [merbanan/rtl_433#2131] [Q&A]
Hi all,

I love this tool on my laptop but it would be so nice to be able to use it on my Android phone / tablet. Did anyone yet try to make an Android apk based on this source?

**Comment by AutomGuy** [comment]
@adewilt8 take a look at here -> [Android](https://github.com/merbanan/rtl_433/issues/892) May be it is helpful

**Comment by Crsarmv7l** [comment]
You can grab an rtl driver here: https://f-droid.org/en/packages/marto.rtl_tcp_andro/
Grab termux: https://f-droid.org/en/packages/com.termux/
and set up a proot distro. You need to launch the driver, but you can do that with an intent when you start termux. Then install rtl_433. Works great for me
